lib/newrelic_httprb/instrumentation.rb

```diff
@@ -0,0 +1,33 @@
+DependencyDetection.defer do
+  named :http_rb
+
+  depends_on do
+    defined?(HTTP) && defined?(HTTP::Client)
+  end
+
+  executes do
+    ::NewRelic::Agent.logger.info 'Installing http.rb instrumentation'
+    require 'new_relic/agent/cross_app_tracing'
+    require 'newrelic_http/wrappers'
+  end
+
+  executes do
+    class HTTP::Client
+      def perform_with_newrelic_trace(request, options)
+        wrapped_request = ::NewRelicHTTP::HTTPRequest.new(request)
+
+        ::NewRelic::Agent::CrossAppTracing.tl_trace_http_request(wrapped_request) do
+          # RUBY-1244 Disable further tracing in request to avoid double
+          # counting if connection wasn't started (which calls request again).
```

Comment:
Yeah, that's what I was suggesting, to use the fork just as a workaround to allow us to test their specs, nothing more :+1: